.github/workflows/snyk.yml

```
@@ -18,10 +18,14 @@ on:
     - cron: '14 6 * * 0' # Random time
   workflow_dispatch:
   
+defaults:
+  run:
+    shell: pwsh
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)